core/arch/x86/opcode.h

```diff
@@ -1387,8 +1387,12 @@ enum {
 };
 
 enum { /* FIXME: vs RAW_OPCODE_* enum */
+       CS_SEG_OPCODE = 0x2e,
```

Comment:
`= RAW_PREFIX_jcc_not_taken`.  (Do not merge the 2 enums in this PR -- even this addition is a little off track for this PR and could be separated.)

Reply:
No, just set to the existing one (hence quoted snippet)

Reply:
Do you want
[..]
CS_SEG_OPCODE = RAW_PREFIX_jcc_not_taken,
[..]
or
not have CS_SEG_OPCODE at all and use RAW_PREFIX_jcc_not_taken in decode_fast
?

Reply:
My original suggestion was trying to be precise with the quoted code. 
 Trying again: `s/0x2e/RAW_PREFIX_jcc_not_taken/` to avoid having 0x2e twice.

Reply:
Done.